### PR TITLE
add org model and get by id methods

### PR DESCRIPTION
**Pull request by kradical**
Added an organization model, split it into resourceinstance and resourcetemplate model. added get by id endpoints so that basic user registration is still properly supported (the client can get all the organizations).

### CODE REVIEW

File: meteor/management/commands/create_organization.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.core.management.base import BaseCommand
+
+from meteor.models import Organization
+
+class Command(BaseCommand):
+    help = "Uses usgs earthquake endpoint to populate the database"
```
Comment (kradical) [inline meteor/management/commands/create_organization.py]: Need to update this description.

File: meteor/views.py [hunk 2]
```diff
@@ -55,19 +58,26 @@ def authenticateuser(request):
...
-    return create_user(request)
+
+# Users
+def user(request):
+    if request.method == 'POST':
+        return create_user(request)
+    if request.method == 'GET':
+        return get_user(request)
+    
+    return JsonResponse({'error': 'WRORNG_METHOD'}, status=405)
```
Comment (kradical) [inline meteor/views.py]: typo WRORNG lul

File: meteor/models.py [hunk 3]
```diff
@@ -5,36 +5,67 @@ class DisasterMap(models.Model):
...
+    """
+    A resource instance. Defined by its template, is specific to one map.
+    """
+    quantity = models.FloatField()
     unit = models.CharField(max_length=120)
     
     location = models.PointField()
 
     disaster_map = models.ForeignKey('DisasterMap', on_delete=models.CASCADE)
+    template = models.ForeignKey('ResourceTemplate', on_delete=models.CASCADE)
```
Comment (ProgrammingJoe) [inline meteor/models.py]: Are we going to make some of our own templates?

Comment (kradical) [inline meteor/models.py]: yes, we could have a script that populates 10-15 but this leaves it open for orgs to add their own.

Comment (ProgrammingJoe) [inline meteor/models.py]: Awesome, that's what I figured you were thinking

**Pull request merged**